app/models/user.rb

```diff
@@ -498,26 +498,30 @@ def number_articles_per_day
     article_count = user.page_views.select(:article_id).distinct.count
     return article_count.round(1) if user.created_at.today?
 
-    (article_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
+    (article_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round
   end
 
   def average_active_time_per_day
-    seconds_spent_reading = user.page_views.sum(:time_tracked_in_seconds).round(1)
+    seconds_spent_reading = user.page_views.sum(:time_tracked_in_seconds).round
     return seconds_spent_reading if user.created_at.today?
 
     (user.page_views.sum(:time_tracked_in_seconds).to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
   end
 
   def number_words_read
     article_words = page_views.joins(:article).select("articles.processed_html").pluck("articles.processed_html")
-    return article_words.sum { |article| article.delete("<p>", "</p>").split.size } if user.created_at.today?
+    article_words.sum { |article| article.delete("<p>", "</p>").split.size }
```

Comment:
fixed an error that prevented from seeing the stat when `user` had been created the same day